nexus_keycode/test/protocols/test_passthrough_uart.py

```diff
@@ -0,0 +1,17 @@
+from unittest import TestCase
+
+import nexus_keycode.protocols.passthrough_uart as uart
+
+
+class TestPassthroughUart(TestCase):
+    def test_passthrough_uart__expected_value_returned(self):
+        """ Test standard input into passthrough_uart and check
+            output"""
+        # 'input_bytes' is actually the 'base key' we are deriving
+        # the uart security key from
+        input_bytes = "\x01" * 14 + "\x43\x51"
+        uart_security_key = (
```

Comment:
Similar to comments above - Python 3 expects bytestrings. You can tell Python 3 you are dealing with bytestrings by prepending `b` to the string, e.g.

```
        input_bytes = b"\x01" * 14 + b"\x43\x51"
        uart_security_key = (
            b"\x12\xe4\x87\x62\x5c\x6b\x88\xf4\x1e\xe4\x0b\x16\xb4\xc9\x84\xf2"
        )
```
![image](https://user-images.githubusercontent.com/1801695/126918738-9596b405-6f55-4829-a157-9fb356c4126e.png)

https://blog.feabhas.com/2019/02/python-3-unicode-and-byte-strings/